### ISSUE TITLE
Custom field not showing up

### ISSUE THREAD

**Issue by stephan-strate** [trilbymedia/grav-plugin-flex-objects#91]
We have added a custom form field in our theme, its working in regular page blueprints.
Adding it to a custom flex object form, it does not even show up.

Maybe this has something to do with flex object being initialized earlier? I could not add flex object definitions in our theme either, needed to add it in a plugin.

**Comment by mahagr** [comment]
Can you give an example for me to work with? Saves some time if I know exactly what you're doing. Thanks.

**Comment by stephan-strate** [comment]
Sure! We have got a standard theme (generated using the devtools).
Created custom form field like this:
![image](https://user-images.githubusercontent.com/19595615/101881488-cc141980-3b94-11eb-9c8b-06cf80a5b898.png)

documentation.html.twig
```twig
{% extends 'forms/field.html.twig' %}

{% block contents %}
    some output
{% endblock %}
```

Added template path in my-theme.php like this:
```
'onTwigTemplatePaths'   => ['onTwigTemplatePaths', 0],
```

```php
public function onTwigTemplatePaths()
{
    $this->grav['twig']->twig_paths[] = __DIR__ . '/templates';
}
```

flex-object form field:
```yaml
documentation:
  type: documentation
```

**Comment by mahagr** [comment]
Yeah, because frontend templates are not being loaded in admin -- they break admin if you do so.

Move fields into `admin/templates` and use `onAdminTwigTemplatePaths` event:

```php
    public function onAdminTwigTemplatePaths(Event $event)
    {
        $paths = $event['paths'];
        $paths[] = __DIR__ . '/admin/templates';
    }
```

**Comment by stephan-strate** [comment]
Got it! Is there a similar strategy to add the flex object blueprints in themes? I got it working in `user/blueprints` and in the plugins blueprints folder, but not in theme right now.

**Comment by phmg701** [comment]
> Got it! Is there a similar strategy to add the flex object blueprints in themes? I got it working in `user/blueprints` and in the plugins blueprints folder, but not in theme right now.

Hello, I had the same question and it now seems fixed in next 1.7.8 release: https://github.com/getgrav/grav/issues/3255
I'll do tests as soon as 1.7.8 gets releases.

Hope this helps!

**Comment by mahagr** [comment]
Yup, the missing blueprints issue has also been fixed, but you need to move the files around (to follow the same blueprints folder structure as in plugins) to enable it.
